### DISCUSSION TITLE
Ability to play older playlists

### DISCUSSION THREAD

**Discussion by XeZey** [ppy/osu#18991] [Ideas] [answered]
Playlists allow a player to look for certain types of maps and then combine them into a pack. This could be for example stream/technical maps and also proves useful outside of std. It would make finding certain types of maps very easy especially in comparison to how one would usually go about finding skillset-specific beatmaps. Having playlists only be available for some amount of time after they were made seems to severely limit this idea.

**Comment by peppy** [comment, marked as answer]
One way you can play them currently is right click -> make copy.

I do agree that a way to "renew" a playlist could be cool

**Reply by XeZey** [reply]
By "older playlists" I was referring to playlists that are older than the category "recently ended", as in there's no way to even see those playlists anymore unless you have played them when they existed.

**Reply by peppy** [reply]
Right. We do also want to add pagination there, and the ability to search.
